pkgs/development/python-modules/lcov-cobertura/default.nix

```diff
@@ -19,7 +20,15 @@ buildPythonPackage rec {
     hash = "sha256-76jiZPK93rt/UCTkrOErYz2dWQSLxkdCfR4blojItY8=";
   };
 
-  build-system = [ setuptools ];
+  postPatch = ''
```

Comment:
This change could really do with a comment linking to the upstream issue.

Reply:
done!